src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+body,
+.navList {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.navList {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+}
+
+.linkAdress {
+  display: flex;
+  text-decoration: none;
+  color: black;
+  padding: 22.5px 0;
+  margin-right: 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.apple {
+  color: #00acdc;
+}
+
+.linkAdress:hover {
+  transition: 1s ;
```

Comment:
transition should work on hover and when cursor leaves element
Also, 1 second it's too long animation, 0.3s is good